app/fg/location-bar/index.js

```diff
@@ -290,16 +289,25 @@ class LocationBar extends LitElement {
     var queryId = ++this.queryIdCounter
     this.inputValue = this.inputValue.trim()
     var finalResults
+    var settings = await bg.beakerBrowser.getSettings()
+    var searchEngine
+    //get search engine to use
+    if (settings.selected_search_engine == -1) {
```

Comment:
The -1 logic here won't be needed after the custom search-engine gets stored to the `search_engines` array, but it _would_ be smart to have DDG as a builtin backup in case the value is bad for some reason.